src/selectors.ts

```diff
@@ -63,7 +63,7 @@
     return config;
   }, [options, extensionId, extensionPointId]);
 
-  const [extensionPoint, isPending] = useAsyncState(async () => {
+  const [extensionPoint, isPending] = useAsyncState(() => {
```

Comment:
Codacy has a fix for the issue: [Functions that return promises must be async.](https://app.codacy.com/gh/pixiebrix/pixiebrix-extension/pullRequest?prid=7747711)
```suggestion
  const [extensionPoint, isPending] = useAsyncState(async () => {
```